pkg/sfu/rtpmunger.go

```diff
@@ -55,170 +53,157 @@ type SnTs struct {
 // ----------------------------------------------------------------------
 
 type RTPMungerState struct {
-	LastSN uint16
-	LastTS uint32
+	ExtLastSN uint32
+	LastTS    uint32
 }
 
 func (r RTPMungerState) String() string {
-	return fmt.Sprintf("RTPMungerState{lastSN: %d, lastTS: %d)", r.LastSN, r.LastTS)
+	return fmt.Sprintf("RTPMungerState{extLastSN: %d, lastTS: %d)", r.ExtLastSN, r.LastTS)
 }
 
 // ----------------------------------------------------------------------
 
-type RTPMungerParams struct {
-	highestIncomingSN uint16
-	lastSN            uint16
-	snOffset          uint16
-	lastTS            uint32
-	tsOffset          uint32
-	lastMarker        bool
-
-	snOffsets          [SnOffsetCacheSize]uint16
```

Comment:
This is the memory saving. The cache size was set at 4K. `RangeMap` uses ranges and stores up to 100 ranges. Each range is about 12 bytes.